app/Models/Trade.php

```diff
@@ -9,16 +9,20 @@ class Trade extends Model
 {
     use HasFactory;
 
-    public function close()
-    {
-        $this->open = false;
-        $this->save();
-    }
```

Comment:
waarom?

Reply:
Deze functie is geschreven tijdens dat ik de migraties gemaakt heb. Later hebben wij besloten de open boolean in de database weg te laten en gewoon te kijken of responder ingevuld is. Deze open value is dus nutteloos en de functie zal niet werken, deze mag dus weg.